openedx/features/specializations/views.py

```diff
@@ -8,4 +15,24 @@ def list_specializations(request):
 
 
 def specialization_about(request, specialization_uuid):
-    return render_to_response('features/specializations/about.html', {})
+    context = DiscoveryClient().get_program(specialization_uuid)
+
+    courses = []
+
+    for course in [c for c in context.get('courses') if c['course_runs']]:
```

Comment:
Let's please make all list comprehension more readable, or let's not use list comprehensions if they are effecting readability and let's please use meaningful names instead of 'c' and 'i'